frontend/src/__tests__/pages/Admin/Subcontractors.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import adminRoutes from "../../../routes/subRoutes/adminRoutes";
+import { MemoryRouter, Routes } from "react-router-dom";
+import { Subcontractors } from "../../../pages/Admin/Subcontractors";
+import { shallow } from "enzyme";
+
+//Mock keycloak.
+jest.mock("@react-keycloak/web", () => ({
+  useKeycloak: () => ({ initialized: true, keycloak: { authenticated: true } }),
+}));
+
+describe("<Subcontractors /> routing", () => {
+  it("renders Subcontractors page when '/admin/contacts/' is hit", () => {
+    render(
+      <MemoryRouter initialEntries={["/admin/contacts/"]}>
+        <Routes key="main">{adminRoutes}</Routes>
+      </MemoryRouter>
+    );
+    shallow(<Subcontractors />);
```

Comment:
I'm not seeing any assertions in this test. Also you need to save the output of shallow to a variable